source/guide_keycloak.rst

```diff
@@ -143,144 +160,38 @@ Setup Backend
+* temporarily delete the web backend using ``userspace web backend del isabell.uber.space``
+* create a directory ``.well-known`` inside your ``/var/www/virtual/isabell/html``
+* create a file ``microsoft-identity-association.json`` in that ``.well-known`` directory with the following contents:
 .. code-block::
- :emphasize-lines: 4
-
- [isabell@stardust ~]$ ps aux
- USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
- isabell   1138  0.0  0.0 219848 17536 ?        Ss   Nov02  11:40 /opt/uberspace/python-venv/bin/python /usr/bin/supervisord -c /etc/supervisord.conf
- isabell   1786  0.4  1.5 1659844 477116 ?      Sl   Nov02 116:23 java -D[Standalone] -server -Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true - Djboss.modules.system.pkgs=org.jboss.byteman
- isabell  23108  0.0  0.0 116912  3556 pts/6    Ss   22:25   0:00 -bash
- isabell  26803  0.0  0.0 155452  1844 pts/6    R+   23:18   0:00 ps aux
- isabell  32155  0.0  0.0 737800 27388 ?        Ss   Nov18   0:11 php-fpm: master process (/opt/uberspace/etc/isabell/php-fpm.conf)
- [isabell@stardust ~]$
-
-And kill the process. Replace ``PID`` by the PID identified in the previous step.
-
-::
-
- [isabell@stardust ~]$ kill -SIGKILL PID
- [isabell@stardust ~]$
-
-Run the upgrade script
-----------------------
-
-.. note:: If you are using a different configuration file than the default one, edit the migration script to specify the new file name.
 
-Now run the script:
+ {
+   "associatedApplications": [
+     {
+       "applicationId": "<your Azure AD application ID>"
+     }
+   ]
+ }
 
-::
+* start the validation in Microsoft's Azure AD console
+* delete the ``.well-known`` directory once complete and recreate the web backend with ``uberspace web backend set [...]``
 
- [isabell@stardust ~]$ cd keycloak-current
- [isabell@stardust keycloak-current]$ bin/jboss-cli.sh --file=bin/migrate-standalone.cli
- *** WARNING ***
 
- ** If the following embed-server command fails, manual intervention is needed.
- ** In such case, remove any <extension> and <subsystem> declarations referring
- ** to the removed smallrye modules from the standalone.xml file and rerun this script.
- ** For details, see Migration Changes section in the Upgrading guide.
- ** We apologize for this inconvenience.
-
- *** Begin Migration ***
- [...]
- *** End Migration ***
- [isabell@stardust keycloak-current]$
-
-Start the daemon again
-----------------------
-
-::
-
- [isabell@stardust ~]$ supervisorctl start keycloak
-  keycloak: started
- [isabell@stardust ~]$
-
-Clean up
---------
-
-Once you made sure had success and everything works as expected you can delete the previous version:
-
-::
-
- [isabell@stardust ~]$ rm -r keycloak-12.0.0
- [isabell@stardust ~]$
-
-
-
-.. _Keycloak: https://www.keycloak.org/downloads
```

Comment:
removing these lines breaks all external links

Reply:
@christianstrauch since you didn't react to this comment, I'm assuming there was no reason to remove these lines. I've re-added them to resolve the broken links.